app/src/testSystem/java/org/mozilla/tv/firefox/engine/CustomContentRequestInterceptorTest.kt

```diff
@@ -20,6 +22,15 @@ import org.robolectric.RobolectricTestRunner
 
 @RunWith(RobolectricTestRunner::class)
 class CustomContentRequestInterceptorTest {
+    @Before
+    fun setup() {
+        // This is setting a internal field just for testing purposes as
+        // WebSettings.getDefaultUserAgent isn't mocked by Roboelectric
+        val defaultUserAgentField = SystemEngine::class.java.getDeclaredField("defaultUserAgent")
+        defaultUserAgentField.isAccessible = true
+        defaultUserAgentField.set(null, "test-ua-string")
```

Comment:
Nit: this field is left mutable for testing, so we don't need to use reflection here.

```kotlin
SystemEngine.defaultUserAgent = "test-ua-string"
```

Reply:
@Baron-Severin that's exactly what I was looking for - thanks!

Reply:
@psymoon sorry that took me so long to find!  I went looking for it yesterday and wasn't able to find it.